mappings/net/minecraft/util/crash/CrashReport.mapping

```diff
@@ -9,6 +9,8 @@ CLASS net/minecraft/class_128 net/minecraft/util/crash/CrashReport
 	FIELD field_1093 cause Ljava/lang/Throwable;
 	METHOD <init> (Ljava/lang/String;Ljava/lang/Throwable;)V
 		ARG 1 message
+		ARG 2 cause
+	METHOD method_24305 dontPanic ()V
```

Comment:
What is this? Please choose a more descriptive name (for a method taking no param and returning void)

Reply:
![image](https://user-images.githubusercontent.com/2484721/77741016-53057e00-7068-11ea-9c24-865a3dca764f.png)